src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/agent-not-starting-windows.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: The agent is not starting in Windows
+type: troubleshooting
+tags:
+  - Infrastructure
+  - Infrastructure monitoring troubleshooting
+  - Troubleshoot infrastructure
+metaDescription: Troubleshooting procedures if the New Relic infrastructure agent is not starting in Windows.
+---
+
+## Problem [#problem]
+
+You installed the New Relic infrastructure agent aon a Windows server and no data is being displayed in the UI.
+When trying to start the service manually from **Windows Services** it fails with an error such as: 
+
+```
+Windows could not start the New Relic Infrastructure Agent service on Local Computer
+Error 1067: The process terminated unexpectedly
+```
+
+## Solution [#solution]
+
+An antivirus or security product might be preventing the New Relic infrastructure agent to be executed as a service. 
+To validate that agent behaves without issues, run the `newrelic-infra.exe` file from the command line as an administrator, and confirm that the host is reporting data as expected. If that's the case, work with your security team to see if the service needs to be added to your allow list, or configured with additional parameters. 
```

Comment:
We could link to [this page](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/windows-installation/zip-manual-install-infrastructure-agent-windows/#agent-directory) where we explain the default path for the "newrelic-infra.exe" file

Reply:
Yes, that sounds great actually! I'll add a link.